src/rust/dbv2/migrations/2023-11-16-103707_market_volume/up.sql

```diff
@@ -0,0 +1,111 @@
+-- Your SQL goes here
+CREATE MATERIALIZED VIEW aggregator.markets_24h_data AS
+    WITH fills AS(
+        SELECT
+            *
+        FROM
+            fill_events
+        WHERE
+            "time" >= CURRENT_TIMESTAMP - '1 day'::interval
+            AND emit_address = maker_address
+    )
+    SELECT
+        market_id,
+        MIN(fills.price) AS min_price_24h,
+        MAX(fills.price) AS max_price_24h,
+        SUM(fills.size) AS base_volume_24h,
+        SUM(fills.size * fills.price) AS quote_volume_24h
+    FROM
+        fills
+    GROUP BY
+        market_id;
+
+
+DROP VIEW api.markets;
+
+
+CREATE VIEW api.markets AS
+    WITH last_fills AS(
+        SELECT
+        DISTINCT ON (market_id)
+            *
+        FROM
+            fill_events
+        WHERE
+            "time" >= CURRENT_TIMESTAMP - '1 day'::interval
+        ORDER BY
+            market_id,
+            txn_version DESC,
+            event_idx DESC
+    ),
+    first_fills AS(
+        SELECT
+        DISTINCT ON (market_id)
+            *
+        FROM
+            fill_events
+        WHERE
+            "time" >= CURRENT_TIMESTAMP - '1 day'::interval
+        ORDER BY
+            market_id,
+            txn_version ASC,
+            event_idx ASC
+    )
+    SELECT
+        m.market_id,
+        m.time AS registration_time,
+        m.base_account_address,
+        m.base_module_name,
+        m.base_struct_name,
+        m.base_name_generic,
+        m.quote_account_address,
+        m.quote_module_name,
+        m.quote_struct_name,
+        m.lot_size,
+        m.tick_size,
+        m.min_size,
+        m.underwriter_id,
+        CASE
+            WHEN r.market_id = m.market_id THEN true
+            ELSE false
+        END AS is_recognized,
+        l.price AS last_fill_price_24hr,
+        (COALESCE(l.price, 1) - COALESCE(f.price, 1)) / COALESCE(f.price, 1) * 100 AS percent_change_24h,
+        l.price - f.price AS nominal_change_24h,
```

Comment:
This is the change in integer prices over 24 hr, whereas prices don't become nominal until coin decimals are accounted for

So how about `AS price_change_24h`?

edit: okay I see that it is nominal to contrast with the percent change field which comes directly before

So instead, how about `AS price_change_24hr`, and the prior one `price_change_as_percent_24hr`